XMLtoMD/GuidelineXmlToMD/Guideline.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+
+
+namespace GuidelineXmlToMD
+{
+    public class Guideline
+    {
+        public string Key { get; set; } = "";
+        public string Text { get; set; } = "";
+
+        public string Severity { get; set; } = "";
+
+        public string Section { get; set; } = "";
+        public string Subsection { get; set; } = "";
+
+        public List<string> Comments { get; set; } = new List<string>();
+
+        public override int GetHashCode()
+        {
+            return Subsection.GetHashCode();
```

Comment:
There is a disconnect between the `GetHashCode` and `Equals` here. These should be acting on the same property.

Reply:
I am trying to think why I did this... It should be the key. Thanks